submit.py

```diff
@@ -24,23 +30,39 @@
 name = Path(args.model_path).parent.name + "-" + Path(args.model_path).stem
 config = saved["config"]
 
+model_type = "temporal_augmentor" if args.model_type.lower() == "t" else "original"
+competition = "germany" if args.competition.lower() == "g" else "south_africa"
+
+if model_type == "temporal_augmentor":
+    assert all([f not in config for f in ['include_bands', "include_cloud", "include_ndvi", "image_size"]])
+
 print(f"Creating: {name}.tar.gz ")
 
 print(config)
 
-if config['competition'] == 'germany':
+if 'germany' in config['competition']:
     pos = '33N_17E_243N'
-elif config['competition'] == 'south_africa':
+    config['competition'] = "germany"
+    assert args.competition.lower() == "g"
+    groundtruth = "/cmlscratch/hkjoo/repo/ai4eo/data/dlr_fusion_competition_germany_test_labels/dlr_fusion_competition_germany_test_labels_33N_17E_243N/crops_test_2019.geojson"
+elif 'south_africa' in config['competition']:
     pos = "34S_20E_259N"
+    config['competition'] = "south_africa"
+    assert args.competition.lower() == "s"
+    groundtruth = None # "Put the path to South Africa's groundtruth test geojson file"
+else:
+    raise NameError
+
+groundtruth = gpd.read_file(groundtruth)
 
 _, reader = load_reader(
     competition=config['competition'],
     satellite=config["satellite"],
     pos=pos,
-    include_bands=config["include_bands"],
-    include_cloud=config["include_cloud"],
-    include_ndvi=config["include_ndvi"],
-    image_size=config["image_size"],
+    include_bands=config["include_bands"] if "include_bands" in config else True,
+    include_cloud=config["include_cloud"] if "include_cloud" in config else True,
+    include_ndvi=config["include_ndvi"] if "include_ndvi" in config else False,
+    image_size=config["image_size"] if "image_size" in config else None,
```

Comment:
These will always be in config won't they?

Reply:
Not the models trained with train_ta.py.  Those four are removed from argparse and hard-coded with booleans (check train_ta.py line 63 through 67).